### PR TITLE
Tests for Webhook sender

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Motivation and context
-  [x] Add tests that verify that the webhook for each event is sent correctly
-  [x] Move all signals sending in one place

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/python/rest_api/test_webhooks_sender.py [hunk 1]
```diff
@@ -96,3 +107,578 @@ def test_webhook_project_update(self):
...
+
+        deliveries, payload = get_deliveries(webhook_id)
+
+        assert deliveries["count"] == 1
+        assert payload["before_update"]["name"] == organizations[org_id]["name"]
+        assert payload["organization"]["name"] == patch_data["name"]
+
+
+@pytest.mark.usefixtures("changedb")
+class TestWebhookPint:
```
Comment (yasakova-anastasia) [inline tests/python/rest_api/test_webhooks_sender.py:615]:
```suggestion
class TestWebhookPing:
```

**Comment by yasakova-anastasia** [comment]
Looks good to me. But I have one question. Why are there no tests for comments?

**Comment by kirill-sizov** [comment]
> Looks good to me. But I have one question. Why are there no tests for comments?

Added tests for comments

**Review by yasakova-anastasia — APPROVED** [review]

**Pull request merged**